src/hooks/useFriction.ts

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 
-import { CSSPairs, getInterpolatorForPair } from '../helpers/pairs';
+import { CSSPairs, getInterpolatorsForPairs } from '../helpers/pairs';
 import { Friction1DParams, friction1D, Controller } from '../animation';
 import { getMaxDistanceFriction } from '../forces';
 
 type UseFrictionArgs = CSSPairs &
-  Omit<Friction1DParams, 'onUpdate' | 'onComplete'>;
+  Omit<Friction1DParams, 'onComplete' | 'onUpdate'>;
 
 export const useFriction = <M extends HTMLElement>({
   from,
   to,
   config,
+  immediate = true,
```

Comment:
I just added these to `useFriction` thus far. I wanted to test things out here before going whole hog and applying them to `useGravity`.